nrf802154_sniffer/nrf802154_sniffer.py

```diff
@@ -63,6 +63,10 @@ class Nrf802154Sniffer(object):
     DLT='802.15.4'
     DLT_NO = 147 if DLT == 'user' else 230
 
+    # USB device identification.
+    NORDICSEMI_VID = 0x1915
+    SNIFFER_802154_PID = 0x0154
```

Comment:
Is PID finally 0x0154 or 0x1540? I think the latter?

Reply:
It's `0x0154` in the firmware I downloaded from this repo. I reserved `0x1540` for the sniffer purpose, but someone changed my reservation to `0x0154`. Don't know who ;)

Reply:
It has been reverted. As I mentioned in my previous comment, the hex file should be changed and this PR accepted now.